.jcheck/conf

```diff
@@ -34,3 +34,8 @@ pattern=^([124-8][0-9]{6}): (\S.*)$
 
 [checks "problemlists"]
 dirs=test/jdk|test/langtools|test/lib-test|test/hotspot/jtreg|test/jaxp
+
+[checks "copyright"]
+files=^(?!LICENSE|license\.txt|.*\.bin|.*\.gif|.*\.jpg|.*\.png|.*\.patch|.*\.wav|.*\.class|.*-header|.*\.jar|).*
```

Comment:
IIUC there are other special files that could be excluded here: .icon, .tiff, .dat

But, again IIUC, this is just an optimization as the check would be "if the file has what appears to be an Oracle copyright, then validate the format". So binary files would not have it and so would not be validated.